### PR TITLE
[TASK] Webpacker 6 Update

### PR DESCRIPTION
**Pull request by michsch**

### CODE REVIEW

File: app/javascript/packs/application.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* 
+Any CSS added to this file or imported from this file, e.g. `@import '../stylesheets/my-css.css'`,
```
Comment (oliverklee) [inline app/javascript/packs/application.css:2]: We now have two `application.css` - one in `app/javascript/` and one in `app/assets/stylesheets/`. Do we need both?

**Review by oliverklee — CHANGES_REQUESTED** [review]
I get the following error after `dip provision` when I run `dip rails s` and request `http://localhost:3000/`:

```
Completed 500 Internal Server Error in 947ms (Allocations: 6100)
 
ActionView::Template::Error (Webpacker can't find application.js in /app/public/packs/manifest.json. Possible causes:
1. You want to set webpacker.yml value of compile to true for your environment
   unless you are using the `webpack -w` or the webpack-dev-server.
2. webpack has not yet re-run to reflect updates.
3. You have misconfigured Webpacker's config/webpacker.yml file.
4. Your webpack configuration is not creating a manifest.
Your manifest contains:
{
}
):
     9:     = stylesheet_link_tag 'application',
    10:       media: 'all',
    11:       data: { 'turbolinks-track': true }
    12:     = javascript_pack_tag 'application', data: { 'turbolinks-track': true }
    13: 
    14:   %body
    15:     %p.notice= notice
```

File: app/javascript/packs/application.css (hunk 1 above)
Comment (michsch) [reply app/javascript/packs/application.css:2]: Done

File: app/packs/entrypoints/application.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+require("@rails/ujs").start()
+require("turbolinks").start()
```
[the remark this answers is not in the document]
Comment (michsch) [reply app/packs/entrypoints/application.js:11]: Done

File: package.json [hunk 3]
```diff
@@ -3,11 +3,20 @@
   "private": true,
   "dependencies": {
     "@rails/ujs": "^6.0.3-4",
-    "@rails/webpacker": "^5.1.1",
-    "turbolinks": "^5.2.0"
+    "@rails/webpacker": "^6.0.0-beta.5",
+    "turbolinks": "^5.2.0",
+    "webpack": "^5.11.0",
+    "webpack-cli": "^4.2.0"
```
[the remark this answers is not in the document]
Comment (michsch) [reply package.json:9]: Done

File: package.json [hunk 4]
```diff
@@ -3,11 +3,20 @@
   "private": true,
   "dependencies": {
     "@rails/ujs": "^6.0.3-4",
-    "@rails/webpacker": "^5.1.1",
-    "turbolinks": "^5.2.0"
+    "@rails/webpacker": "^6.0.0-beta.5",
+    "turbolinks": "^5.2.0",
+    "webpack": "^5.11.0",
```
[the remark this answers is not in the document]
Comment (michsch) [reply package.json:8]: Done

File: package.json [hunk 5]
```diff
@@ -3,11 +3,20 @@
   "private": true,
   "dependencies": {
     "@rails/ujs": "^6.0.3-4",
-    "@rails/webpacker": "^5.1.1",
-    "turbolinks": "^5.2.0"
+    "@rails/webpacker": "^6.0.0-beta.5",
```
[the remark this answers is not in the document]
Comment (michsch) [reply package.json:6]: Done

**Comment by oliverklee** [comment]
@michsch Let's do a parallel new checkout and installation together during our next meetup.

**Review by oliverklee — CHANGES_REQUESTED** [review]
Now we do not get the crash anymore, but some JavaScript still does not get loading.

Please try adding a book and then deleting it again - there JavaScript is needed to change the HTTP action to DELETE.

File: log/.bash_history [hunk 6]
```diff
@@ -0,0 +1,12 @@
+bin/webpack --debug-webpacker
```
Comment (oliverklee) [inline log/.bash_history:1]: This file should not be versioned.

File: package.json [hunk 7]
```diff
@@ -3,11 +3,22 @@
   "private": true,
   "dependencies": {
     "@rails/ujs": "^6.1.3-1",
-    "@rails/webpacker": "^5.2.1",
-    "turbolinks": "^5.2.0"
+    "@rails/webpacker": "^6.0.0-beta.5",
```
Comment (oliverklee) [inline package.json:6]: This now needs to be `6.0.0-beta.7` (without the `^`) to be in sync with the Gem version.

File: package.json [hunk 8]
```diff
@@ -3,11 +3,22 @@
...
+    "turbolinks": "^5.2.0",
+    "webpack": "^5.11.0",
```
Comment (oliverklee) [inline package.json:8]: 5.36.0 now is the latest version.

File: package.json [hunk 9]
```diff
@@ -3,11 +3,22 @@
...
+    "webpack-cli": "^4.2.0"
```
Comment (oliverklee) [inline package.json:9]: 4.6.0 now is the latest version.

**Review by oliverklee — CHANGES_REQUESTED** [review]
Looks very good - and best of all, the JS in the FE now works!

We still need to get rid of the bash history file and up the minimum versions in the `package.json`.

**Pull request merged**